.github/CODEOWNERS

```diff
@@ -173,13 +188,18 @@
 /core/templates/dev/head/pages/topic_viewer/ @aks681
 
 # Topics and skills dashboard project.
-/core/controllers/topics_and_skills_dashboard.py @aks681
+/core/controllers/subtopic_viewer*.py @aks681
+/core/controllers/topics_and_skills_dashboard*.py @aks681
 /core/templates/dev/head/domain/topics_and_skills_dashboard/ @aks681
 /core/templates/dev/head/pages/topics_and_skills_dashboard/ @aks681
 
 
 # Infrastructure.
+/core/controllers/cron*.py @seanlip
```

Comment:
@seanlip, should I create a different section for `corn, jobs and task` or we can place this over here (inside Infrastructure)? &

Reply:
I think infrastructure is fine.